language-server/test/dotty/tools/languageserver/CompletionTest.scala

```diff
@@ -1,22 +1,146 @@
 package dotty.tools.languageserver
 
+import org.junit.Assert.{assertTrue, assertFalse}
 import org.junit.Test
-import org.eclipse.lsp4j.CompletionItemKind
+import org.eclipse.lsp4j.CompletionItemKind._
 
 import dotty.tools.languageserver.util.Code._
 
 class CompletionTest {
 
   @Test def completion0: Unit = {
     code"class Foo { val xyz: Int = 0; def y: Int = xy$m1 }".withSource
-      .completion(m1, Set(("xyz", CompletionItemKind.Field, "Int")))
+      .completion(m1, Set(("xyz", Field, "Int")))
   }
 
   @Test def completionWithImplicitConversion: Unit = {
     withSources(
       code"object Foo { implicit class WithBaz(bar: Bar) { def baz = 0 } }",
       code"class Bar",
       code"object Main { import Foo._; val bar: Bar = new Bar; bar.b${m1} }"
-    ) .completion(m1, Set(("baz", CompletionItemKind.Method, "=> Int")))
+    ) .completion(m1, Set(("baz", Method, "=> Int")))
+  }
+
+  @Test def importCompleteClassWithPrefix: Unit = {
+    withSources(
+      code"""object Foo { class MyClass }""",
+      code"""import Foo.My${m1}"""
+    ).completion(m1, Set(("MyClass", Class, "Object{...}")))
+  }
+
+  @Test def ImportCompleteClassNoPrefix: Unit = {
+    withSources(
+      code"""object Foo { class MyClass }""",
+      code"""import Foo.${m1}"""
+    ).completion(m1, results => {
+      val myClass = ("MyClass", Class, "Object{...}")
```

Comment:
This "Object{...}" thing is weird, we should either not display anything, or list the class parents.

Reply:
I've changed it so that we show the type for terms (like we do currently), and the fully qualified name for types. What do you think?